src/api/utils.test.js

```diff
@@ -143,3 +148,223 @@ describe('checkData', () => {
     expect(() => utils.checkData({})).toThrow();
   });
 });
+
+describe('useWebSocket', () => {
+  const kind = 'TaskRun';
+  it('should handle Created events', () => {
+    const queryClient = getQueryClient();
+    const webSocket = getWebSocket();
+
+    const wrapper = ({ children }) => (
+      <APIWrapper queryClient={queryClient} webSocket={webSocket}>
+        {children}
+      </APIWrapper>
+    );
```

Comment:
How about we extract it to a function like:

```
// return a wrapper function.
function createWrapper(queryClient, WebSocket) {
  return ({children}) => <APIWrapper queryClient={queryClient} webSocket={webSocket}>{children}</APIWrapper>;
}
```

And then call `createWrapper(queryClient, WebSocket)` in renderHook instead?

Reply:
I like this, I've extracted it to a function as you suggested 👍